ocaml/xapi-cli-server/cli_operations.ml

```diff
@@ -7113,6 +7113,60 @@ let host_reset_server_certificate _printer rpc session_id params =
 let host_emergency_clear_mandatory_guidance _printer rpc session_id _params =
   Client.Host.emergency_clear_mandatory_guidance ~rpc ~session_id
 
+let host_updates_show_available fd _printer rpc session_id params =
+  do_host_op rpc session_id ~multiple:false
+    (fun _ host ->
+      let host = host.getref () in
+      let pool = get_pool_with_default rpc session_id params "uuid" in
+      let master = Client.Pool.get_master ~rpc ~session_id ~self:pool in
+      let master_address =
+        Http.Url.maybe_wrap_IPv6_literal
+          (Client.Host.get_address ~rpc ~session_id ~self:master)
+      in
+      let open Xmlrpc_client in
+      let transport =
+        SSL
+          ( SSL.make () ~verify_cert:(Stunnel_client.pool ())
+          , master_address
+          , !Constants.https_port
+          )
+      in
```

Comment:
This function is executed on the pool coordinator. Even if you run the command on another pool member, it will be forwarded. So we do not need a network connection and definitely not TLS. The easiest would be to connect to the unix socket. More efficient would be to short-circuit the HTTP handler, as is done for the `rpc` function (see the top of xapi_cli.ml and `register_callback_fns` in xapi.ml)